### ISSUE TITLE
docu Verlinkung sieht komisch aus

### ISSUE THREAD

**Issue by MaraHochstein** [MaraHochstein/EPMAdatatools#45]
https://hezel2000.github.io/mag4/microprobe/data-access/epmatools-data-access.html
![grafik](https://github.com/user-attachments/assets/6f6a414d-74c1-41ef-9ca6-99b350627fb0)

unten sieht es etwas komisch aus (auch unter "Welcome")

**Comment by Hezel2000** [comment]
Ja, da stimmt was mit den GDPR Einstellungen nicht. Ich bin an den Seiten schon seit ein paar Tagen dran, es gibt da einige Baustellen. Es wird sich wohl noch bis in den Januar ziehen, bis ich das alles erledigt bekomme.

**Comment by Hezel2000** [comment]
Ich konnte den Fehler nun endlich beheben. Das neue GDPR Formular sieht zwar nicht mehr so slick aus, aber dafür funktioniert es wieder. D.h., als nächstes kann ich mich an die Überarbeitung der Hilfeseiten machen.
